File: docs-site/nodejs/subscribe.js
// Returns a boolean
var subscribed = await client.on("chat.room1", (data) => {
    console.log(data);
})

// -------------------------------------------------
// $data is a json
{
    id: "<MESSAGE ID>",
    topic: "<TOPIC MATCHING TOPIC / WILDCARD TOPIC>",
    data: <Actual message as string, number or json>
}

File: docs-site/python/subscribe.py
def callback_fn(data):
    print(data)

# Returns a boolean
subscribed = await client.on("chat.room1", callback_fn)

# -------------------------------------------------
# $data is a dict
{
    "id": "<MESSAGE ID>",
    "topic": "<TOPIC MATCHING TOPIC / WILDCARD TOPIC>",
    "data": <Actual message as string, number or json>
}